src/NativeScript/TNSRuntime+Inspector.mm

```diff
@@ -29,6 +29,8 @@
         : _messageHandler(Block_copy(handler)) {
     }
 
+    virtual ConnectionType connectionType() const override { return ConnectionType::Local; };
```

Comment:
This change in the inspector comes from https://github.com/WebKit/webkit/commit/f7d7d3e933a74f63bbe276f8794efab20a552fe6. We are using the inspector with a local connection so a ```ConnectionType::Local``` return value is OK.